### DISCUSSION TITLE
[Feature proposal]: -webkit-text-stroke support

### DISCUSSION THREAD

**Discussion by shaevtchac** [tailwindlabs/tailwindcss#11935] [Ideas]
Hi
I'd like to propose one feature.
`stroke` tool exists and works fine for SVG elements but not for normal text which is being used much more often. 
`-webkit-text-stroke` is widely supported in browsers (95%) [caniuse](https://caniuse.com/?search=-webkit-text-stroke)

`stroke` should default to `-webkit-text-stroke` in my opinion but I understand that's not going to happen so I suggest creating new tool like `textstroke` which will work in a similar way to stroke but for normal elements.

It would be even better if you could extend `stroke `tool to include normal elements making it generating different code depending where it is used. E.g. `stroke-white` and `stroke-1` used on (or inside) SVG element would generate `stroke: white;` and `stroke-width: 1;`, but when used on normal text it results in `-webkit-text-stroke-color: white;` and `-webkit-text-stroke-width: 1px;`
That would also solve #3501.
Whaddaya think? 😄

**Comment by lukewarlow** [comment]
Is there a syntax difference between the two? If not I don't see why it can't just be added to the util. Failing that autoprefixer / lightningcss could handle it?

**Reply by shaevtchac** [reply]
Syntax would be exactly the same and that is why I think would be better just extend the currently existing `stroke ` tool instead of creating a new one. Let the framework figure out what to do freeing user from thinking what tool to use where. Stroke is a stroke : )
